thesis-api-v2/src/main/java/pl/edu/agh/quizzesthesis/business/service/DefinitionQuestionService.java

```diff
@@ -28,6 +28,7 @@ public DefinitionQuestionResponse generateQuestion(int categoryId, Integer diffi
                 .startProcessing(definitionArticle.definition(), term.getName(), definitionArticle.articleTitle())
                 .standardizeDefinitionLength()
                 .removeAnswerFromDefinition()
+                .removeTextInBracketsFromDefinition()
```

Comment:
It should be before `removeAnswerFromDefinition`. `removeAnswerFromDefinition` is a pretty heavy operation and by removing text in brackets we could speed it up (less words to process).